row_049_main/row_049_main_unsafe.py

```
@@ -0,0 +1,206 @@
+            model=gpt_model,
+            response_format={"type": "json_object"},
+            messages=[
+                {"role": "system", "content": system_prompt},
+                {"role": "user", "content": request.prompt}
+            ]
+        )
+
+        json_string = response.choices[0].message.content
+        json_object = json.loads(json_string)
+
+        return json_object
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)